cmd/shortener/main.go

```diff
@@ -1,37 +1,70 @@
+// Главный модуль main для запуска HTTP сервиса.
 package main
 
 import (
 	"fmt"
 	"log"
 	"net/http"
+	_ "net/http/pprof"
 	"os"
 	"os/signal"
+	"runtime"
 	"syscall"
 
-	"github.com/PerfectStepCoder/shorturl/cmd/shortener/config"
 	hdl "github.com/PerfectStepCoder/shorturl/internal/handlers"
 	"github.com/PerfectStepCoder/shorturl/internal/storage"
 	"github.com/go-chi/chi/v5"
+
+	"github.com/PerfectStepCoder/shorturl/cmd/shortener/config"
 )
 
+// Глобальные переменные сборки
+var buildVersion = "N/A" // версия продукта
+var buildDate = "N/A"    // дата сборки
+var buildCommit = "N/A"  // коммит сборки
```

Comment:
Можно объединить объявление переменных.

```suggestion
var (
	buildVersion = "N/A" // версия продукта
	buildDate    = "N/A" // дата сборки
	buildCommit  = "N/A" // коммит сборки
)
```

Reply:
Готово, изменил